src/Feature.cpp

```diff
@@ -42,21 +43,43 @@ void Feature::Load(json& o_json)
 	SI_Error rc = ini.LoadFile(ini_path.c_str());
 
 	if (rc < 0) {
-		if (!FeatureIssues::IsObsoleteFeature(GetShortName()))
+		if (FeatureIssues::IsObsoleteFeature(GetShortName())) {
+			// Handle obsolete features
+			version = "obsolete";  // Set version so it shows error color in UI
+			failedLoadedMessage = std::format("{} is an obsolete feature that has been removed.", GetShortName());
+			FeatureIssues::FeatureFileInfo fileInfo = FeatureIssues::GetFeatureFileInfo(GetShortName());
+			FeatureIssues::AddFeatureIssue(GetShortName(), "N/A", failedLoadedMessage,
+				FeatureIssues::FeatureIssueInfo::IssueType::OBSOLETE, fileInfo, "");
+		} else {
 			logger::info("{} failed to load, feature disabled", ini_filename);
+
```

Comment:
Same here. I don't think you understand the code. The ini is missing because they didn't install or deleted it. Why is that an issue to raise?